src/script/standard.h

```diff
@@ -73,8 +73,19 @@ class CNoDestination {
     friend bool operator<(const CNoDestination &a, const CNoDestination &b) { return true; }
 };
 
-struct WitnessV0ScriptHash : public uint256 {};
-struct WitnessV0KeyHash : public uint160 {};
+struct WitnessV0ScriptHash : public uint256
+{
+public:
+    WitnessV0ScriptHash() : uint256() {}
+    WitnessV0ScriptHash(const uint256& in) : uint256(in) {}
+};
+
+struct WitnessV0KeyHash : public uint160
+{
+public:
+    WitnessV0KeyHash() : uint160() {}
```

Comment:
You could instead inherit the ctors from uint160/uint256 to shortcut the construction from ```WitnessV0KeyHash(uint160(foo)``` to ```WitnessV0KeyHash(foo)```:
```c++
using uint160::uint160;
```

Reply:
Cool, done.